pom.xml

```diff
@@ -40,7 +40,11 @@
     <org.sweble.wikitext.version>3.1.9</org.sweble.wikitext.version>
 
     <spring.version>6.2.6</spring.version>
+    <commons.codec.version>1.18.0</commons.codec.version>
+    <commons.compress.version>1.27.1</commons.compress.version>
+    <commons.io.version>2.19.0</commons.io.version>
```

Comment:
Needs a runtime test (was bumped from 1.4). Don't know if swc works with 2.x or if it has sufficient tests to cover that area.

Reply:
Think it's ok. It seems I upgraded the original lib a years ago to 2.x without issues. So should be fine.

Reply:
Can confirm it works at runtime: tested in a separate project :)